flutter/lib/src/integrations/frames_tracking_integration.dart

```diff
@@ -0,0 +1,73 @@
+import '../../sentry_flutter.dart';
+import '../binding_wrapper.dart';
+import '../frames_tracking/sentry_delayed_frames_tracker.dart';
+import '../frames_tracking/span_frame_metrics_collector.dart';
+import '../native/sentry_native_binding.dart';
+
+class FramesTrackingIntegration implements Integration<SentryFlutterOptions> {
+  FramesTrackingIntegration(this._native);
+
+  final SentryNativeBinding _native;
+  SentryFlutterOptions? _options;
+  PerformanceCollector? _collector;
+  SentryWidgetsBindingMixin? _widgetsBinding;
+
+  @override
+  Future<void> call(Hub hub, SentryFlutterOptions options) async {
+    _options = options;
+
+    void abortInitWith({required String reason}) {
+      options.logger(SentryLevel.debug,
+          '$FramesTrackingIntegration is disabled: $reason.');
+    }
+
+    if (!options.enableFramesTracking) {
+      return abortInitWith(reason: 'option enablesFrameTracking is false');
+    }
+
+    if (options.tracesSampleRate == null && options.tracesSampler == null) {
+      return abortInitWith(reason: 'tracing is disabled');
+    }
+
+    final widgetsBinding = options.bindingUtils.instance;
+    if (widgetsBinding == null ||
+        widgetsBinding is! SentryWidgetsBindingMixin) {
+      return abortInitWith(
+          reason:
+              'incompatible binding, SentryFlutterWidgetsBinding has not been instantiated');
```

Comment:
I would add the instruction in the log message (Please, use `SentryFlutterWidgetsBinding.ensureInitialized()` instead of `FlutterWidgetsBinding.ensureInitialized()`)
And I'd make it a `warning`, or at least `info` level